src/app/pages/challenges/challenge-view/challenge-view.component.ts

```diff
@@ -14,22 +20,77 @@ import { PageTitleService } from '@shared/page-title/page-title.service';
   styleUrls: ['./challenge-view.component.scss'],
 })
 export class ChallengeViewComponent implements OnInit {
-  challenge$!: Observable<Challenge>;
+
+  @Input() selected!: boolean;
+  @Output() selectedChange = new EventEmitter<boolean>();
+
+  // challenge$!: Observable<Challenge>;
+  challenge!: Challenge;
+  personList: Person[] = [];
+  orgList: Organization[] = [];
+  progressValue: number = 0;
 
   constructor(
     private router: Router,
     private route: ActivatedRoute,
     private challengeService: ChallengeService,
+    private organizationService: OrganizationService,
+    private personService: PersonService,
     private pageTitleService: PageTitleService
   ) {}
 
+
   ngOnInit(): void {
-    this.challenge$ = this.route.params.pipe(
-      switchMap((params) => this.challengeService.getChallenge(params.id))
+    // this.challenge$ = this.route.params.pipe(
+    //   switchMap((params) => this.challengeService.getChallenge(params.id))
+    // );
+    // this.challenge$.subscribe(challenge => {
+    //   this.pageTitleService.setTitle(challenge.name);
+    // });
+    this.route.params.pipe(
+      switchMap((params) => this.challengeService.getChallenge(params.id)),
+      tap(challengeRes => {
```

Comment:
```suggestion
      tap(challenge => {
```

Reply:
>Ideally align horizontally to the left the left side of a section title and the left side of its content. Unless this is part of the Card initial design.

It was card's original design, but I will change to make them align.